ast/cinaps/shortcuts.mli

```diff
@@ -0,0 +1,13 @@
+(** Type for shortcuts from description types to their parent record types.
+    E.g. from [expression_desc] to [expression]. *)
+type t
+
+(** Returns true if the given record has a `_desc` field, ie if it is at the
+    end of a shortcut. *)
```

Comment:
I don't really understand the second part of this sentence. What about just:

```ocaml
(** Returns true if the given record has a [XXX_desc] field. *)
```

BTW, note that we must use `[` rather than ``` ` ``` in doc comments.